features/comments_and_kudos/hidden_comments.feature

```diff
@@ -65,3 +65,26 @@ Feature: Comment hiding
       And I view the work "Popular Fic" with comments
     Then I should see "A suspicious comment"
       And I should not see "This comment has been hidden by an admin."
+
+  Scenario Outline: Embedded images in hidden comments are replaced with their URLs.
+    Given I am logged in as "author"
+    And I post the work "Popular Fic"
+    And I am logged out
```

Comment:
Can you convert this into a single `Given the work "Popular Fic"` step? It runs a bit faster than logging in and posting manually